### DISCUSSION TITLE
error while doing the evaluation of the trained model

### DISCUSSION THREAD

**Discussion by YuanbinLiu** [ACEsuit/mace#123] [Q&A] [answered]
Does anyone have an idea about this error? I encountered it while using a trained potential to predict a dataset.

python ~/mace/scripts/eval_configs.py \
                                             --configs="test.extxyz" \
                                             --model="MACE_model_run-123.pt" \ 
                                             --output="output.extxyz"

Traceback (most recent call last):
  File "/Users/yuanbinliu/software/mace/scripts/eval_configs.py", line 151, in <module>
    main()
  File "/Users/yuanbinliu/software/mace/scripts/eval_configs.py", line 88, in main
    z_table = utils.AtomicNumberTable([int(z) for z in model.atomic_numbers])
AttributeError: 'dict' object has no attribute 'atomic_numbers'

**Comment by davkovacs** [comment, marked as answer]
You should use a .model file not a .pt file (which is just a checkpoint, but not the actual model) 
You can easily convert between the two if you want to evaulate a specific checkpoint by re-running the training with restart latest option setting max num epochs to 0. This works if the specific .pt checkpoint is in the checkpoints directory.
